lms/templates/discussion/_underscore_templates.html

```diff
@@ -571,7 +571,7 @@ <h1 class="home-title">${course.display_name_with_default}</h1>
     </script>
 </%def>
 
-${secondaryAction("edit", "pencil", _("Edit"))}
+${secondaryAction("edit", "pencil-square-o", _("Edit"))}
```

Comment:
NOTE: syncing up icons used for "editing" actions (currently just discussions + now notes) in the LMS.